### PR TITLE
The document for setup Istio with ICP

### PR DESCRIPTION
**Pull request by skeeey**
The document for setup Istio with ICP

### CODE REVIEW

**Comment by gyliu513** [comment]
@skeeey please sign the CLA first.

**Comment by skeeey** [comment]
I signed the CLA

**Comment by skeeey** [comment]
I signed it!

**Comment by skeeey** [comment]
I signed it!

**Comment by linsun** [comment]
/lgtm

@frankbu PTAL

File: content/docs/setup/kubernetes/quick-start-icp/index.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+title: Quick Start with IBM Cloud Private
+description: How to quickly setup Istio using IBM Cloud Private.
+weight: 21
+keywords: [kubernetes,icp]
+---
+
+Follow these instructions to install and run Istio in the
```
Comment (frankbu) [inline content/docs/setup/kubernetes/quick-start-icp/index.md]: in the -> in

File: content/docs/setup/kubernetes/quick-start-icp/index.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+## Prerequisites
+
+- You need to have an available IBM Cloud Private cluster. Otherwise, you can follow [Installing IBM Cloud Private-CE](https://www.ibm.com/support/knowledgecenter/SSBS6K_2.1.0.3/installing/install_containers_CE.html) to create an IBM Cloud Private cluster.
+
+## Deploy Istio via the Catalog module
+
+- Log in to the **IBM Cloud Private** console.
+- Click `Catalog` on the right side of the navigation bar.
+- Click `Filter` on the right side of the search box and select the `ibm-charts` checking box.
```
Comment (frankbu) [inline content/docs/setup/kubernetes/quick-start-icp/index.md]: checking box -> checkbox

File: content/docs/setup/kubernetes/quick-start-icp/index.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    caption="IBM Cloud Private - Istio Installation"
+    >}}
+
+## Upgrade or Rollback
+
+- Log in to the **IBM Cloud Private** console.
+- Click the menu button on the left side of the navigation bar.
+- Click `Workloads` and select `Helm Releases`.
+- Find the installed Istio using its release name.
+- Click `Action` link and select `upgrade` or `rollback`.
```
Comment (frankbu) [inline content/docs/setup/kubernetes/quick-start-icp/index.md]: Click `Action` and select ...

File: content/docs/setup/kubernetes/quick-start-icp/index.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    caption="IBM Cloud Private - Istio Upgrade or Rollback"
+    >}}
+
+## Uninstalling
+
+- Log in to the **IBM Cloud Private** console.
+- Click the menu button on the left side of the navigation bar.
+- Click `Workloads` and select `Helm Releases`.
+- Find the installed Istio using its release name.
+- Click `Action` link and select `delete`.
```
Comment (frankbu) [inline content/docs/setup/kubernetes/quick-start-icp/index.md]: Click `Action` and select ...

File: content/docs/setup/kubernetes/quick-start-icp/index.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    >}}
+
+- Click `ibm-istio` in the right panel.
+
+{{< image width="100%" ratio="50%"
+    link="./istio-catalog-2.png"
+    caption="IBM Cloud Private - Istio Catalog"
+    >}}
+
+- (Optional) Change the Istio version using `CHART VERSION` drop-down.
```
Comment (frankbu) [inline content/docs/setup/kubernetes/quick-start-icp/index.md]: using the

**Comment by linsun** [comment]
@skeeey can you please open a separate PR to address @frankbu 's comments?   I didn't expect the merge bots to merge with his rejection.

**Comment by Rachael-Graham** [comment]
I'll grab these grammatical changes in #2485

**Pull request merged**